lib/act.js

```diff
@@ -9,8 +9,9 @@ var call;
 var config;
 var widget;
 var cache;
+var debug = require('debug')('fh-mbaas-api:act');
```

Comment:
I'd argue for simplicity over having a wrapper (just require and use it).

Maybe a good approach is forking the debug module & adding timestamps?
I think the current approach in the PR is a great improvment on what was there before, and could be built upon.
@wojta interested in your thoughts.